housekeeper/include.py

```diff
@@ -31,13 +33,13 @@ def include_version(global_root: str, version_obj: models.Version, hardlink:bool
         else:
             os.symlink(file_obj_path.resolve(), new_path)
         log.info(f"linked file: {file_obj.path} -> {new_path}")
-        file_obj.path = str(new_path).replace(f"{global_root_dir}/", '', 1)
+        file_obj.path = str(new_path).replace(f"{global_root_dir}/", "", 1)
```

Comment:
I have not actually touched this code, it is black. I have tried to avoid touching the code as much as possible since it would make this PR grow forever 🙄 . But I added this suggestion anyway